pkg/apis/pipeline/v1alpha1/verificationpolicy_types.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// +genclient
+// +genclient:noStatus
+// +genreconciler:krshapedlogic=false
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VerificationPolicy defines the resources and corresponding authorities to verify.
```

Comment:
docstrings like this could probably use a bit more detail, for example what is being verified. instead of commenting on each docstring I'll observe there seems to be a general pattern around the docstrings in this file (and maybe others) of not providing any extra info in the docstring beyond what the code says for the type themselves - the fields themselves seem to have much better docstrings